### ISSUE TITLE
[FEATURE REQUEST] Location-based working set for fixpoints

### ISSUE THREAD

**Issue by lucaneg** [lisa-analyzer/lisa#316]
**Description**
Sorting nodes to process w.r.t. their location will reduce the time needed to compute fixpoints:
```java
while (x > 0)
    x--;
f(x);
```
After the while guard is processed both, `x--` and `f(x)` are added to the ws. `x--` is analyzed first, and the while guard is placed back into the ws **at the beginning**. The analysis of `f(x)` will be deferred until the whole loop stabilizes, and will thus be analyzed only once.
